src/controlsystem.jl

```diff
@@ -1,30 +1,97 @@
-struct ControlSystem{N,T,Fsys<:Function,Fbound<:Function}
+abstract type ControlSystem end
+
+struct ControlSystemGrowth{VT<:SVector,F1<:Function,F2<:Function} <: ControlSystem
     tstep::Float64
-    sysnoise::SVector{N,T}
-    measnoise::SVector{N,T}
-    sys_map::Fsys
-    bound_map::Fbound
+    sysnoise::VT
```

Comment:
Since sysnoise is used in growthbound_map, it should be a SVector{N,T} with same N,T as the first argument of sys_map and growthbound_map.
In fact, I struggled a bit to decide how to choose the parametric types of controlsystem (and am not sure the last version is optimal). Do not hesitate to make changes :)